src/DataSource.ts

```diff
@@ -48,21 +49,16 @@ export class DataSource extends DataSourceWithBackend<GitHubQuery, GithubDataSou
 
     const res = await this.query(query).toPromise();
 
-    if (!res || !res.data || res.data.length < 0) {
+    if (!(res?.data?.length)) {
       return [];
     }
-
-    const view = new DataFrameView(res.data[0] as DataFrame);
-
-    const results = view.map(item => {
-      return {
-        title: `${request.annotation.name} - ${annotation.queryType}`,
-        time: item[annotation.timeField || 'name'],
-        text: item[annotation.field || 'name'],
-      };
+    return getAnnotationsFromFrame(res.data[0], {
+      field: {
+        // title: `${request.annotation.name} - ${annotation.queryType}`,
```

Comment:
Do you want to have the same title for everything in the query result?

Reply:
I don't see why not. After using this for a bit, the ` - ${annotation.queryType}` thing caused a lot of stuttering "Open Issues - Issues" for exxample.